### DISCUSSION TITLE
error with quick start templates

### DISCUSSION THREAD

**Discussion by sohail-hussain** [Budibase/budibase#5327] [Help]
Running budibase in a corporate network where internet access is unavailable
There is a message "There was a problem loading quick start templates."

based packages/server/src/utilities/fileSystem/index.js i was able to find https://prod-budi-templates.s3-eu-west-1.amazonaws.com/manifest.json
  

Is there a way to 

1. download the quick start templates using my browser, and save them in a directory
OR
2. specify an outbound proxy

**Comment by melohagan** [comment]
Hi @sohail-hussain 

There's no way to do this at the moment. 

If there's a particular template you're interested in, then you could create a template app with a Budibase Cloud account outside of the corporate network, export it, and then import it within your corporate network, if this is permissible.

**Reply by mojoaar** [reply]
@melohagan Not to hijack the thread - but is there any way to turn of the check for templates/updates, to stop the massive amount of toast notifications when you are behind a company proxy?
![image](https://user-images.githubusercontent.com/21560785/175538561-3f4d327e-be28-4a33-96ef-96b8bb57e6c1.png)

**Reply by melohagan** [reply]
Hi @mojoaar Not currently. If you're building the docker images yourself, you could comment out this line in `create.svelte`
![Screenshot 2022-06-24 at 15 10 37](https://user-images.githubusercontent.com/101575380/175553307-deaec890-0411-4b7f-9c8d-1ecd58b2924d.png)

**Reply by jamespharvey20** [reply]
> Hi @mojoaar Not currently. If you're building the docker images yourself, you could comment out this line in `create.svelte`

I'd also like to disable the error message.  I downloaded `docker-compose.yaml` and `.env`, then ran `docker-compose up`.  Where would `create.svelte` be?  I don't see it in the directory I downloaded those files to, and I don't see it anywhere in `/var/lib/docker/`.  (Docker newbie here.)  Or, would I need to be using the Budibase CLI Install method?

**Reply by nerveband** [reply]
Bumping up also for the same question when using the Docker image.

**Reply by burkeazbill** [reply]
And here's another bump... A simple checkbox to disable, or a field to provide an alternate location for the json would be quite nice for those of us using the pre-built docker image. Even an environment setting for those would work well.
